### ISSUE TITLE
Request payload is missing in Chrome immediately after a fetch in RTK Query

### ISSUE THREAD

**Issue by alex-vukov** [reduxjs/redux-toolkit#3716]
This is a actually caused by this bug in Chromium https://bugs.chromium.org/p/chromium/issues/detail?id=1058404
Here in the RTK Query repo https://github.com/reduxjs/redux-toolkit/blob/master/packages/toolkit/src/query/fetchBaseQuery.ts#L269
`request.clone()` is used which triggers the bug. This bug causes any Playwright or Puppeteer test which checks an outgoing fetch request's body to fail in Chrome.
The reason why I am logging this issue in your repo is because the Chromium bug has been open for 3 years and is still not resolved so I thought maybe there is an alternative way to copy the request contents in your code which doesn't use the `clone()` function. This would be a workaround until the Chromium bug is fixed. I myself can try to contribute with a PR.

**Comment by alex-vukov** [comment]
From what I tested it seems pretty straightforward to change:
```
    const request = new Request(url, config)
    const requestClone = request.clone()
```

to

```
    const request = new Request(url, config)
    const requestClone = new Request(url, config)
```

**Comment by markerikson** [comment]
Fixed in 1.9.6, I believe.

**Comment by alex-vukov** [comment]
I tested and it works. Thank you!
